deploy-manage/remote-clusters/_snippets/eck_rcs_expose.md

```diff
@@ -0,0 +1,36 @@
+When the remote cluster server is enabled, ECK automatically creates a Kubernetes service named `<cluster-name>-es-remote-cluster` that exposes the server internally on port `9443`:
+
+```sh
+quickstart-es-remote-cluster       ClusterIP      None             <none>         9443/TCP            4h13m
+```
+
+To allow other clusters running outside your Kubernetes environment to connect, you must expose this service externally. As of ECK {{version.eck}}, you cannot customize the service that ECK generates for the remote cluster interface, but you can create your own `LoadBalancer` service, `Ingress` object, or use another method available in your environment.
```

Comment:
do we need to specify "as of eck 3.2"?  if this ever changes that change will be reflected here

Reply:
In ECK 3.3 (planned) we will be supporting the configuration of this service directly within the ECK-managed manifest (functionality added [here](https://github.com/elastic/cloud-on-k8s/pull/8892)).

I'm updating this to include a couple of `applies switch` tabs, but if you prefer we can hide all this until 3.3 is released. Let me know your thoughts.